### PR TITLE
docs(i): Update contributors

### PR DESCRIPTION
**Pull request by nasdf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -479,5 +479,6 @@ DefraDB's code is released under the [Business Source License (BSL)](licenses/BS
 - Orpheus Lummis ([@orpheuslummis](https://github.com/orpheuslummis))
 - Fred Carle ([@fredcarle](https://github.com/fredcarle))
 - Islam Aliev ([@islamaliev](https://github.com/islamaliev))
+- Keenan Nemetz ([@nasdf](https://github.com/nasdf))
```
Comment (AndrewSisley) [inline README.md:482]: thought: Way overdue :laughing: - however, do we really need this section?  Contributors are listed in github, and overtime manually growing this list will not scale well.

**Review by AndrewSisley — APPROVED** [review]
LGTM :grin:

File: README.md (hunk 1 above)
Comment (nasdf) [reply README.md:482]: could probably rename it to maintainers since there's a larger difference between contributors and active maintainers

Comment (AndrewSisley) [reply README.md:482]: Might be a good idea, although that would mean removing Orpheus from the list too

**Pull request merged**